### PR TITLE
Added JavaFX functionality to the program

### PR DESCRIPTION
**Pull request by ZingBing**
This PR adds JavaFX requirements and a few changes to main.java, as well as a class file to be able to run JavaFX. This allows us to run our program inside of a java window instead of needing to launch it in the browser, and makes the app seem a lot more standalone and easier to run for the client.

### CODE REVIEW

**Comment by ZingBing** [comment]
Also I noticed that the cover image on the program no longer works when using JavaFX, I am not sure if it has changed regardless of my addition to the project, or if the way the program is running now makes it unable to pull from the internet. We should look into that regardless though and maybe make a way to use a static image or occasionally try to pull an updated roster photo.

File: app/src/main/java/com/mu_bball_stats/WebBrowser.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    @Override
+    public void start(Stage primaryStage) {
+        // Create a WebView component
+        WebView webView = new WebView();
+
+        // Load the URL served by your Javalin server
+        webView.getEngine().load("http://localhost:7070");
+
+        // Create a Scene and add the WebView to it
+        Scene scene = new Scene(webView, 800, 600);
```
Comment (jpappel) [inline app/src/main/java/com/mu_bball_stats/WebBrowser.java:19]: we should look into initializing the scene to the display size

**Review by jpappel — APPROVED** [review]
Good work, thank you for stepping up to the plate by finding a new technology. 👍 :)

**Comment by jpappel** [comment]
> We should look into that regardless though and maybe make a way to use a static image or occasionally try to pull an updated roster photo.

I am not entirely sure, but I believe as part of the build process we can have gradle download an image. That might strike a good balance between packaging a splash image and hyperlinking to one.

**Pull request merged**